tests/infrastructure/taints-and-tolerations.go

```diff
@@ -227,12 +234,13 @@ func getDeploymentsForPods(virtClient kubecli.KubevirtClient, pods []k8sv1.Pod)
 	Expect(err).NotTo(HaveOccurred())
 
 	var result []types.NamespacedName
-	for _, deployment := range allDeployments.Items {
+	for i := range allDeployments.Items {
+		deployment := allDeployments.Items[i]
 		selector, err := metav1.LabelSelectorAsSelector(deployment.Spec.Selector)
 		Expect(err).NotTo(HaveOccurred())
 
-		for _, pod := range pods {
-			if selector.Matches(labels.Set(pod.Labels)) {
+		for y := range pods {
```

Comment:
nit: for loop index variables are named in general like this: {'i', 'k', 'l', ..} - was just odd to me when reading though, will not block on this.

Reply:
Addressed this in the latest push